src/resources/ProductListingConfiguration/ProductListingConfigurationInterfaces.ts

```diff
@@ -0,0 +1,248 @@
+import {DeprecatedPaginated} from '../InternalBaseInterface.js';
+import {
+    FacetRequestOptions,
+    FieldOperatorType,
+    FieldValueType,
+    FilterTypeEnum,
+    RankingTypeEnum,
+    RestFacetRequest,
+} from '../index.js';
+
+export const isRankingTypeEnum = (value: unknown): value is RankingTypeEnum =>
+    value === RankingTypeEnum.BOOST || value === RankingTypeEnum.BURY;
+
+export const isFilterTypeEnum = (value: unknown): value is FilterTypeEnum =>
+    value === FilterTypeEnum.INCLUDE || value === FilterTypeEnum.EXCLUDE;
+
+export enum SortOrderEnum {
+    ASC = 'ASC',
+    DESC = 'DESC',
+}
+
+export enum SortPropertyEnum {
+    NAME = 'NAME',
+    REVENUE = 'REVENUE',
+    CONVERSION_RATE = 'CONVERSION_RATE',
+    BOUNCE_RATE = 'BOUNCE_RATE',
+}
+
+export enum RuleVariantEnum {
+    BASELINE = 'BASELINE',
+    CAMPAIGN = 'CAMPAIGN',
```

Comment:
probably need to be lowercase?

Reply:
That's how it was in the PLM, I'll look how platform-client usually handle it.

Reply:
yeah a lot of the enums were uppercase and i've been gradually changing them to lowercase when the backend rejected them